frontend/src/AddSeries/SeriesMonitoringOptionsPopoverContent.js

```diff
@@ -35,6 +35,11 @@ function SeriesMonitoringOptionsPopoverContent() {
         data="Monitor all episodes of the latest season and future seasons"
       />
 
+      <DescriptionListItem
+        title="Specials"
```

Comment:
I think `Monitor Specials` with a description that indicates it will only affect special episodes would be better.

Reply:
This is reworded to Monitor specials with an explanation